DirectProgramming/DPC++FPGA/Tutorials/GettingStarted/fpga_template/src/fpga_template.cpp

```diff
@@ -0,0 +1,78 @@
+#include <iostream>
+
+// oneAPI headers
+#include <CL/sycl.hpp>
+#include <sycl/ext/intel/fpga_extensions.hpp>
+
+using namespace sycl;
```

Comment:
avoid the `using namespace` as much as possible in the new samples, please use `sycl::` explicitly below

Reply:
ok

Reply:
if that's the new style :)

Reply:
That's what we try to do for all new samples as:
1/ this is usually the recommended practice
2/ this helps the users understand from which namespace each object comes from